src/ui/components/StepDots.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import { StepDots } from 'ui/components/StepDots'
+import { ColorsEnum } from 'ui/theme'
+
+describe('<StepDots />', () => {
+  it('should render properly', () => {
```

Comment:
c'est bien de spliter les tests pour mettre en évidence ce qui est tester.

Je pense que 3 tests feraient l'affaire:

- should render properly on first position 
- should render properly on middle position
- should render properly on last position